guides/v2.2/coding-standards/technical-guidelines.md

```diff
@@ -761,6 +761,18 @@ class SampleEventObserverThatModifiesInputs
 
 15.8. Frequently update the third-party libraries used in the project/component to eliminate known vulnerabilities.
 
+15.9. LFI protection.
```

Comment:
Please include abbreviation explanation, as in "13. Command line interface (CLI)".